### PR TITLE
feat: adding tools section

### PR DESCRIPTION
**Pull request by paulmowat**
adding tools section and tiny helpers

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hello @paulmowat thank you for your contribution.

Can you instead add the new topic Tools under Others? Also, add a link to the new topic in the table of content

**Comment by paulmowat** [comment]
Hi @shahednasser, No problem. I've moved tools to it's own section under Others and added to the table of contents. thanks 😄

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ To add a resource or contribute, please check out our contribution guide [here](
   - [Open Source](#open-source)
   - [Standard Template Library](#standard-template-library)
   - [Programming Courses Free Online](#programming-courses-free-online)
+- [Tools](#tools)
```
Comment (shahednasser) [inline README.md:63]: Add indentation for it to be placed under Others

File: README.md [hunk 2]
```diff
@@ -611,3 +608,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 - [How to run graphics.h in VS Code](https://github.com/ullaskunder3/Solution-to-graphics.h)
 - [Top 5 VS Code extensions for web developers](https://jobin.hashnode.dev/top-5-vs-code-extension-for-web-developers-in-2021)
+
+## Tools
```
Comment (shahednasser) [inline README.md:612]: Add an extra #

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paulmowat) [reply README.md]: Thats it fixed. Misunderstood what you meant with under Others. Sorry about that 😄

File: README.md (hunk 2 above)
Comment (paulmowat) [reply README.md]: done

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**